.github/workflows/check-pull-request.yml

```diff
@@ -0,0 +1,26 @@
+name: Check Branch
+on: [push]
+jobs:
+  deploy:
```

Comment:
@bengimbel what do you think about renaming this job to `test`?

Reply:
Sounds good to me @gabrieltaylor